pkg/plugins/mellanox/mellanox_plugin.go

```diff
@@ -206,3 +211,21 @@ func (p *MellanoxPlugin) nicHasExternallyManagedPFs(nicPortsMap map[string]sriov
 	}
 	return false
 }
+
+// nicNotConfiguredByOperator returns true if one of the ports(interface) of the NIC is not configured by operator
+func (p *MellanoxPlugin) nicNotConfiguredByOperator(nicPortsMap map[string]sriovnetworkv1.InterfaceExt) bool {
+	existPfStatusOnStore := false
+	for _, iface := range nicPortsMap {
+		_, exist, err := p.helpers.LoadPfsStatus(iface.PciAddress)
+		if err != nil {
+			log.Log.Error(err, "failed to load PF status from disk", "address", iface.PciAddress)
+			continue
+		}
+		if exist {
+			log.Log.V(2).Info("PF configured by the operator", "interface", iface)
+			existPfStatusOnStore = true
+			break
```

Comment:
```suggestion
			return false
```

can we get rid of the `existPfStatusOnStore` variable and early exit here?

Reply:
right!